pkg/sql/distsqlrun/backfiller_test.go

```diff
@@ -135,7 +164,16 @@ CREATE UNIQUE INDEX vidx ON t.test (v);
 		// Work completed on a span; ["o", "p"] complete.
 		{Key: roachpb.Key("q"), EndKey: roachpb.Key("r")},
 	}
-	got := tableDesc.Mutations[0].ResumeSpans
+
+	job, err = registry.LoadJob(ctx, jobID)
+
+	if err != nil {
+		t.Fatal(errors.Wrapf(err, "can't find job %d", jobID))
+	}
+
+	details = job.Record.Details.(jobs.SchemaChangeDetails)
+
+	got := details.ResumeSpanList[0].ResumeSpans
```

Comment:
same here. It will be nice to have some accessor functions that get/set the resume span on the jobs table.